styx-common/src/test/java/com/spotify/styx/util/CachedSupplierTest.java

```diff
@@ -67,7 +67,7 @@
 
   @Before
   public void setUp() throws Throwable {
-    when(delegate.get()).thenReturn(INITIAL_VALUE);
+    lenient().when(delegate.get()).thenReturn(INITIAL_VALUE);
```

Comment:
This stub is not used by all test cases (but most of them require it). Instead of pushing it down to each test case, let's make it `lenient`.